server/models/users.js

```diff
@@ -41,6 +81,37 @@ class User {
       },
     });
   }
+
+  static async refreshToken(rToken) {
```

Comment:
This could simply just be. 
```suggestion
  static async refreshToken(token) {
```

Reply:
Overall this looks good. A few minor comments

The `package-lock.json` is empty run `npm install` to repopulate it. 

The steps in the parent read me are misleading since they mention a script `dbSetup.sql` which is not in the repo. 

You are also lacking a `.env.example` file which shows other people what a sample `.env` 
should look like running the project.